.github/workflows/dev.yml

```
@@ -325,6 +332,7 @@ jobs:
       - name: "pinact"
         id: "pinact"
         continue-on-error: true
+        env: *lint-auth
         env:
```

YAMLlint (1.37.1)

[error] duplication of key "env" in mapping

(key-duplicates)